qcodes/utils/ndim_analysis/dataset/doNd.py

```diff
@@ -286,7 +286,7 @@ def _handle_plotting(
     return res
 
 
-def _create_plots(data: DataSet) -> Tuple[DataSet, list, list]:
+def get_plots(data: DataSet) -> Tuple[DataSet, list, list]:
```

Comment:
I am not very happy with this name, because it suggests that the plots already exist, and it does not communicate well that it will generate plots on disk, if specified.
Maybe we can simply call it `plot`, which is well defined in the context of the module and is very short.
We should also add all parameters that can be passed into `plot_dataset`.

Reply:
Sure. The name was just a placeholder until such a review.